web/index.html

```diff
@@ -61,8 +90,14 @@
       };
 
       async function onload() {
+        const mapInstance = new mapLibreGL.Map({
+          container: "mapElement",
+          style: "https://tiles.stadiamaps.com/styles/outdoors.json",
+        });
```

Comment:
Pending the outcome of the other discussion, it would be nice to configure the map similarly, with a geolocate control, auto-geolocation on load, center, and pitch.